zipline/finance/blotter.py

```diff
@@ -33,30 +34,36 @@
 
 log = Logger('Blotter')
 
-class Blotter(object):
 
-    def __init__(self):
-        self.transact = transact_partial(VolumeShareSlippage(), PerShare())
```

Comment:
before moving things back, it wasn't being used.

after I added things back, I thought it was too indirect, and would rather explicitly set slippage and commission functions.